WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Payments Menu/InPersonPaymentsMenuViewModel.swift

```diff
@@ -94,6 +103,7 @@ class InPersonPaymentsMenuViewModel: ObservableObject {
         self.dependencies = dependencies
         self.payInPersonToggleViewModel = payInPersonToggleViewModel
         self.simplePaymentsNoticePublisher = PassthroughSubject<SimplePaymentsNotice, Never>().eraseToAnyPublisher()
+        self.orderViewModel = .init(siteID: siteID)
```

Comment:
Related to the question above, I'm not sure of potential ramifications of instantiating a new editable order here, and if this would case miss-use in the future.